### ISSUE TITLE
Use ONNX model in ML.Net

### ISSUE THREAD

**Issue by cherrex** [dotnet/machinelearning#4856]
Hello,
I created and trained a model in a keras.
The basis is a simple and standard task for the classification of irises. The CSV file looks like this

SepalLengthCm, SepalWidthCm, PetalLengthCm, PetalWidthCm, Species
5.1,3.5,1.4,0.2, iris-setosa

Model in keras
model = Sequential ()
model.add (Dense (128, input_dim = 4, activation = 'relu'))
model.add (Dense (Y.shape [1], activation = 'softmax'))
model.compile (loss = 'categorical_crossentropy', optimizer = 'adam', metrics = ['categorical_accuracy'])

I understand that this problem can be solved without keras using ML.NET, but the principle of using keras models in ML.NET is important to me.
Next I save the model to ONNX
onnx_model = keras2onnx.convert_keras (model, model.name)
onnx.save_model (onnx_model, 'model_.onnx')

Tell me how in the future, I can use this model in ML.NET, for predictions?

Thanks in advance.

**Comment by gvashishtha** [comment]
Take a look at the [ApplyOnnxModel(https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.onnxcatalog.applyonnxmodel?view=ml-dotnet) method and at a working sample [here](https://github.com/dotnet/machinelearning-samples/blob/master/samples/csharp/end-to-end-apps/ObjectDetection-Onnx/OnnxObjectDetection/ML/OnnxModelConfigurator.cs#L27)

**Comment by frank-dong-ms** [comment]
@cherrex does Gopal's suggestion work for you? Close the issue now, feel free to reopen if you have further questions, thanks.
